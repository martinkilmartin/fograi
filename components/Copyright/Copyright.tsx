import { BRAND } from '@constants/CONTENT';

const Copyright = (): JSX.Element => (
  <p>{`Â© ${new Date().getFullYear()} ğŸ‡®ğŸ‡ª ${BRAND} ğŸ‡¨ğŸ‡¦`}</p>
);

export default Copyright;
