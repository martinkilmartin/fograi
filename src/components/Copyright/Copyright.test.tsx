import { BRAND } from '@constants/CONTENT'
import renderer from 'react-test-renderer'
import Copyright from './Copyright'

test('Copyright displays current year and brand', () => {
  const copyrightRenderer = renderer.create(<Copyright />)
  const tree = copyrightRenderer.toJSON()
  expect(tree).toMatchSnapshot()
  const copyrightInstance = copyrightRenderer.root
  expect(copyrightInstance.findByType('p').children[0]).toEqual(
    `Â© ${new Date().getFullYear()} ðŸ‡®ðŸ‡ª ${BRAND} ðŸ‡¨ðŸ‡¦`
  )
})
