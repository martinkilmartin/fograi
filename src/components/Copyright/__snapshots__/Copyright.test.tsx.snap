// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Copyright displays current year and brand 1`] = `
<p>
  Â© 2021 ğŸ‡®ğŸ‡ª FÃ³graÃ­ ğŸ‡¨ğŸ‡¦
</p>
`;
